### DISCUSSION TITLE
Capability to assign mapping to LFO parameters on TCP or automation item params

### DISCUSSION THREAD

**Discussion by bodnaar** [helgoboss/helgobox#1370] [Ideas]
Hello.
I might have missed how to do this, though I watched the videos.
I'd like to use a high shelf in reaEQ, and modulate the frequency parameter. I tried two ways: LFO button at TCP, or creating Automation item on track envelope. I couldn't learn the cycles or amplitude param from the automation item, neither could I learn the LFO settings. The mapping always points to the High Shelf param itself, and when I automate (with LFO or automation item), it just doesn't respond to my knob. I wouldn't like to modify the freq by hand anyway, just the LFO freq or amplitude.

Screenshot provided. I hope I could explain the case clearly. Any suggestions?

![realearn](https://github.com/user-attachments/assets/9f1106f0-cfa6-4638-8bb1-123bc39c8151)

**Comment by helgoboss** [comment]
There is no target for controlling automation item or parameter modulation settings. If you want, create a feature request (or turn this discussion into one by changing the type to "Ideas").

**Reply by bodnaar** [reply]
I'll do that. Thank you for the quick elaboration :)

**Reply by bodnaar** [reply]
Anyway, I wondered if I can use compartment params in control transformation? I could then output a parametric function directly to the eq param, not the automation!

**Reply by helgoboss** [reply]
Not yet. There's #488. Oh, and there's #1320.
